setup.py

```diff
@@ -19,7 +19,7 @@
         "docopt >= 0.6.2",
         "flask >= 0.10.1",
         "Flask-Caching >= 1.4.0, < 1.8.0",
-        "Flask-Cors >= 2.1.0",
+        "Flask-Cors >= 2.1.0, < 4",
```

Comment:
Perhaps we should add a comment here, to explain why the `< 4` is needed.

Reply:
Please let me know if https://github.com/cisagov/ncats-webd/pull/38/commits/62d8efb911de4e5b80d47697eb5909607b02cc2e scratches that itch.